js/controller.js

```diff
@@ -49,6 +50,7 @@ function handleGamePlay(event) {
 //This will select the card to play
 var selectPlayer = function(){
   if(Player.allPlayers[0].nextTurn === true){
+    // todo: 'event' does not exist in this function, you need to pass it in
     var cardPlayed = event.target;
```

Comment:
It *looks* like you are trying to reference 'event' from within the 'handleGamePlay' function where this function (selectPlayer) is called from. Probably a mistake from cutting and pasting code.